.pre-commit-config.yaml

```diff
@@ -0,0 +1,14 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+  - repo: https://github.com/norwoodj/helm-docs
+    rev: v1.13.1
+    hooks:
+      - id: helm-docs-container
+        args:
+          # Make the tool search for charts only under the `example-charts` directory
+          - --chart-search-root=gpt-survey-summarizer
```

Comment:
**issue:** This argument is changing the search root of the tool. We need to be able to generate the README for multiple helm charts. I think we could this option instead:
```
  -g, --chart-to-generate strings                          List of charts that will have documentation generated. Comma separated, no space. Empty list - generate for all charts in chart-search-root
```